src/Widgets/ContactFormWidget/ContactFormWidgetEditingConfig.js

```diff
@@ -12,12 +12,22 @@ Scrivito.provideEditingConfig('ContactFormWidget', {
       title: 'Submit button text',
       description: 'Default: send message',
     },
+    isTransparent: {
+      title: 'Background color',
```

Comment:
Smelly. Diverging name and title indicate one of the two should be changed.
Imagine a later refactoring trying to add a third bg color 🙀